sdk/eventhub/event-hubs/src/managementClient.ts

```diff
@@ -293,7 +293,7 @@ export class ManagementClient extends LinkEntity {
       }
 
       const sendRequestOptions: SendRequestOptions = {
-        times: options.retryOptions && options.retryOptions.retryCount,
+        times: options.retryOptions && options.retryOptions.maxRetries,
```

Comment:
Update `times` -> `maxRetries` in retry config before passing it to `retry` method.